### ISSUE TITLE
Add user preference to change Wildcard value display for J, Q, K

### ISSUE THREAD

**Issue by blackshadowshade** [buttonmen-dev/buttonmen#2833]
In #2722, it was noted that some users may prefer to see 11, 12, and 13 instead of J, Q, K. This would require a new user preference.

**Comment by TheOrgg** [comment]
Also 20 instead of Joker.

**Comment by blackshadowshade** [comment]
I disagree, Joker needs to stay as Joker so that it's obvious why there's no 14, 15, ..., especially in dropdowns with fire.

**Comment by TheT113** [comment]
An idea I thought of and wanted to search here to see if it was mentioned:

Instead, why not have it show the value on hovering, something like "Wildcard (Queen of Hearts, worth 12)".

I understand that works counter to how dice are currently displayed in the hover, but I think it'd be a good compromise.

Alternately, leave out the "Queen of Hearts" part, and also maybe there's a better word than "worth".

**Comment by blackshadowshade** [comment]
I like the idea of not creating a new user preference, and this compromise seems to be both reasonable and useful.

**Comment by nkmcalli** [comment]
I opened a related ticket just for the text changes (similar to what @TheT113 suggested: https://github.com/buttonmen-dev/buttonmen/issues/2922
